app/src/main/java/org/fossasia/openevent/app/module/event/create/CreateEventFragment.java

```diff
@@ -154,7 +155,8 @@ public void showProgress(boolean show) {
 
     @Override
     public void onSuccess(String message) {
-        ViewUtils.showSnackbar(binding.getRoot(), message);
+        Toast.makeText(getActivity(), message, Toast.LENGTH_SHORT).show();
+        getActivity().finish();
```

Comment:
Rather than closing in `onSuccess` create a new method in view contract called `close` and call it in the presenter, so that we can test the behaviour